Dshop/apps/products_catalogue/api_urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from .api_views import CartAPIView
+
+
+urlpatterns = [
+    path("add/<int:id>/<int:quantity>/", CartAPIView.as_view(), name="api_add_to_cart"),
```

Comment:
This is not following REST convention. 
REST is using HTTP methods type to do changes, so it should be one item, just different HTTP methods, no?
You already have different, proper handlers in CartAPIView.

Reply:
done